<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportCbDialog</class>
 <widget class="QDialog" name="ExportCbDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>导出到回调接口</string>
  </property>
  <widget class="QPlainTextEdit" name="consoleTextEdit">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>30</y>
     <width>331</width>
     <height>521</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>121</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>过滤条件</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>101</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="allAudioRadioButton">
       <property name="text">
        <string>录制所有音频</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">filterButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="allButMeAudioRadioButton">
       <property name="text">
        <string>录制他人音频</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">filterButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="selectAudioRadioButton">
       <property name="text">
        <string>录制所选音频</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">filterButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="selectVideoRadioButton">
       <property name="text">
        <string>录制所选视频</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">filterButtonGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>270</y>
     <width>271</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>视频列表</string>
   </property>
   <widget class="QListWidget" name="videoList">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>271</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>74</width>
     <height>35</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>当前录制时间</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="timeLabel">
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(85, 85, 255);</string>
      </property>
      <property name="text">
       <string>00:00:00</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>77</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="startButton">
      <property name="text">
       <string>开始</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="stopButton">
      <property name="text">
       <string>停止</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="filterButtonGroup"/>
 </buttongroups>
</ui>
